test/loginTest.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Created by austin on 30/04/2017.
+ */
+'use strict';
+
+const assert = require('chai').assert;
+const fixtures = require('./fixtures.json');
+const kujiraService = require('./kujira-service.js');
+const user = require('./../../kujira-credentials.json');
```

Comment:
as discussed, probably best to change the path of this file to be in the same directory as the test, and also to leave a stub file with stub values in the file, so the user will simply have to navigate to this file and replace the placeholder stub values with their own credential values